src/traces/parcats/defaults.js

```diff
@@ -0,0 +1,122 @@
+/**
+* Copyright 2012-2018, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Lib = require('../../lib');
+var hasColorscale = require('../../components/colorscale/has_colorscale');
+var colorscaleDefaults = require('../../components/colorscale/defaults');
+var handleDomainDefaults = require('../../plots/domain').defaults;
+var handleArrayContainerDefaults = require('../../plots/array_container_defaults');
+
+var attributes = require('./attributes');
+var mergeLength = require('../parcoords/merge_length');
+
+function handleLineDefaults(traceIn, traceOut, defaultColor, layout, coerce) {
+
+    if(traceIn.line) {
+        coerce('line.shape');
+    }
```

Comment:
why are we looking for `traceIn.line` and not just always `coerce('line.shape')` so we always have a default in `fullData`?

Reply:
Done in eba1b4a